pkg/workload/workload.go

```diff
@@ -416,6 +416,16 @@ func applyResourceTransformations(input corev1.ResourceList, transforms map[core
 	output := make(corev1.ResourceList)
 	for inputName, inputQuantity := range input {
 		if mapping, ok := transforms[inputName]; ok {
+			// If MultiplyBy is specified, multiply the input quantity by
```

Comment:
Can you add some unit tests for this feature?

Reply:
looks like ResourceTransformations already have some examples at a quick glance (https://github.com/kubernetes-sigs/kueue/blob/a10c41cd4e5153c12ee01c08ae18bb6e84c6991a/pkg/workload/workload_test.go#L375)